src/pkg/packaging/windows/vscomponents/generatenupkg.ps1

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+param(
+    [Parameter(Mandatory=$true)][string]$MsiPath,
+    [Parameter(Mandatory=$true)][string]$MsiVersion,
+    [Parameter(Mandatory=$true)][string]$NuspecFile,
+    [Parameter(Mandatory=$true)][string]$NupkgFile,
+    [Parameter(Mandatory=$true)][string]$Architecture,
+    [Parameter(Mandatory=$true)][string]$ComponentName,
+    [Parameter(Mandatory=$true)][string]$ComponentFriendlyName,
+    [Parameter(Mandatory=$true)][string]$ProjectUrl
+)
+
+$RepoRoot = Convert-Path "$PSScriptRoot\..\..\..\..\..\.."
```

Comment:
Doesn't this script exist elsewhere already?

Reply:
Not in this repo, other nupkgs created here either use `Pack` or the packaging tools package from Arcade.

Reply:
Ah its in core-sdk

Reply:
I've cleaned this up by passing in the 'BinDir' to the script - so the script doesn't have to do any directory calculations.